### PR TITLE
Travis ci

### PR DESCRIPTION
**Pull request by thehamzarocks**

### CODE REVIEW

File: .github/workflows/google_java_format.yml [hunk 1]
```diff
@@ -3,8 +3,8 @@ name: Format
 
 on:
   pull_request:
-    branches:
-      - master
+    branches: [master]
+    types: [edited, opened, reopened]
```
Comment (thehamzarocks) [inline .github/workflows/google_java_format.yml:7]: Dunno why these changes come up

Comment (daltonfury42) [reply .github/workflows/google_java_format.yml:7]: This is fine, but if you want the pr to be clean of old commits, i think you can merge origin master into the branch.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+language: java
+jdk:
+  - oraclejdk11
+
+script:
+  - cd complete
+  - mvn package
+  - zip -r latest *
```
Comment (daltonfury42) [inline .travis.yml:8]: I've always wondered, is it just a standard to zip all the files? Can't we just take the jar?

File: .github/workflows/google_java_format.yml (hunk 1 above)
Comment (daltonfury42) [reply .github/workflows/google_java_format.yml:7]: never mind, we can merge this and close the other pr?
